RELEASE_NOTES.md

```diff
@@ -15,6 +15,7 @@
   - Up. Payment method type: **mealVoucher_FR_groupeup**.
   - Natixis. Payment method type: **mealVoucher_FR_natixis**.
   - Sodexo. Payment method type: **mealVoucher_FR_sodexo**.
+- Added `CardNumberValidator`, `CardExpiryDateValidator` and `CardSecurityCodeValidator` classes to make validation functionality for the corresponding fields available to public.
```

Comment:
Maybe let's mention at the start that this is for API only merchants with a custom card component?